packages/cdktf-cli/bin/cmds/helper/check-environment.ts

```diff
@@ -55,24 +55,26 @@ export async function checkEnvironment() {
 }
 
 export async function verifySimilarLibraryVersion() {
+  if (process.env.DISABLE_VERSION_CHECK) {
```

Comment:
Should we allow `"false"` to not disable the version check?
```suggestion
  if (process.env.DISABLE_VERSION_CHECK && process.env.DISABLE_VERSION_CHECK !== "false") {
```

Reply:
I read through every one of our `process.env` calls today and none have a check like this. We have one for logging that checks if it's === false. I believe we should overhaul what we currently have and check if it's set to 1 or true, but I'd rather do it globally after this merged

Reply:
fine with me